content/docs/user-guide/basic-concepts/dependency.md

```diff
@@ -3,6 +3,7 @@ name: Dependency
 match: [dependency, dependencies, depends, input]
 tooltip: >-
   A file or directory (possibly tracked by DVC) recorded in the `deps` section
```

Comment:
recorded -> specified 

in general, also too low level and not very informative

Files (e.g. code, data), directories (e.g. datasets), parameters that stage (e.g. train a model) depends on ...